overlord/snapstate/backend_test.go

```diff
@@ -206,6 +206,14 @@ func (f *fakeSnappyBackend) RemoveSnapData(info *snap.Info) error {
 	return nil
 }
 
+func (f *fakeSnappyBackend) RemoveSnapSharedData(info *snap.Info) error {
+	f.ops = append(f.ops, fakeOp{
+		op:   "remove-snap-shared-data",
+		name: info.MountDir(),
```

Comment:
Yeah, it's nice for testing that a given operation happens on a given version of the snap. We may want to leave this as `MountDir()`, then.